app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -202,6 +202,14 @@ def build_metric_id(type, resource, metric_id)
       "#{type}I~R~[#{resource[:middleware_server][:feed]}/#{resource[:nativeid]}]~#{type}T~#{metric_id}"
     end
 
+    def self.update_alert(*args)
```

Comment:
I have the same question. Is this just adding the plumbing for updating and will be completed in a future PR?